intellij/src/saros/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -339,25 +351,31 @@ public void run() {
   }
 
   /**
-   * Creates an empty stub module with the given name and given base path in the given project.
+   * Creates an empty base module with the given name, base path, and module type in the given
+   * project.
    *
-   * <p>The created module has the module type {@link IntelliJProjectImpl#RELOAD_STUB_MODULE_TYPE}
-   * which allows us to easily identify it as stub.
+   * <p>The created module does not contain any further configuration. Need configuration options
```

Comment:
Needed ?

Reply:
Hmm sorry I meant "Needed" instead of "Need". Maybe my English is bad and "Need" is correct although I hardly remember to even read such a sentence.

Reply:
No, "Needed" is indeed correct. Will change it.